src/components/Bullet.tsx

```diff
@@ -66,6 +66,7 @@ const Bullet = ({
         {glyph || (showContexts ? (leaf ? '◦' : '▹') : leaf ? '•' : '▸')}
         <span
           className={classNames('glyph-bg', {
+            expanded: !leaf,
```

Comment:
This line doesn't make sense, but maybe it's just the naming.

- A `leaf` is a thought with no children. It can't be expanded.
- A non-leaf is a thought with children. It can be expanded or collapsed, based on the result of `expandThoughts` with the cursor position.